assets/js/components/wp-dashboard/wp-dashboard-unique-visitors.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * WPDashboardUniqueVisitors component.
+ *
+ * Site Kit by Google, Copyright 2019 Google LLC
```

Comment:
This might as well be updated to 2020, but really we should just search + replace them all to 2021 in a few weeks so 🤷🏻‍♂️

Reply:
I think we should open an issue for the same.